public/maps/india.json

```diff
@@ -4358,7 +4358,7 @@
             ]
           ],
           "type": "Polygon",
-          "properties": {"ST_NM": "Chandigarh"}
+          "properties": {"st_nm": "Chandigarh"}
```

Comment:
Curious, why are there modifications in the map files?

Reply:
Yes for district maps key was in small letters and for state it was in capital for I made that changes for Maintaining consistency

Reply:
Oh alright, I think this might end up solving some map issues actually! 

Could you rather submit these map changes in a different PR since it'd be easier for us to track down the changes?

Reply:
@jeremyphilemon #429 is raised that will seprate map cleanup
as this PR depends upon it once that is merged this can be merged as well

*Hopefully, if that PR won't create any conflict*

Reply:
Haha thank you so much for this PR and the other PR! I can see a lot of issues getting solved in the maps are with the latter.

With respect to localization, I'll be pushing that next! Can't wait to incorporate this into the website! :)